<ui version="4.0" >
 <class>account</class>
 <widget class="QDialog" name="account" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Set account</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Enter a SIP account to make calls</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="e_user" >
     <property name="text" >
      <string>username</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Domain</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="e_domain" >
     <property name="text" >
      <string>iptel.org</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="e_pwd" >
     <property name="text" >
      <string>password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>account</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>207</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>account</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>207</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel" >
     <x>216</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
